apps/pie-microsite/src/assets/styles/components/_list.scss

```diff
@@ -0,0 +1,50 @@
+@use '../fozzie-config' as f;
+
+.c-list {
+    list-style: none;
+    padding-left: 0;
+}
+
+.c-list--ordered {
+    counter-reset: item;
+
+    .c-list-item {
+        counter-increment: item;
+        margin-bottom: f.$spacing-02;
+        @include f.font-size('body-l');
+        display: flex;
+
+        @include f.media('>=wide') {
+            align-items: center;
+        }
+
+        &:before {
+            content: counter(item);
+            @include f.font-size('body-s');
+            font-weight: f.$font-weight-bold;
+            margin-right: f.$spacing-03;
+            width: 25px;
+            height: 25px;
+            background: f.$color-container-inverse;
+            border-radius: 100%;
+            color: f.$color-white;
+            display: flex;
+            align-items: center;
+            justify-content: center;
+            flex: 0 0 auto;
+        }
+    }
+}
+
+.c-list--pill {
+    display: flex;
+    gap: 8px;
+
+    .c-list-item {
+        @include f.font-size('body-s');
+        background-color: f.$color-container-strong;
+        padding: 2px 4px;
+        border-radius: 8px;
```

Comment:
same here regarding the `spacing` thing

Reply:
As there's not a token for these values I've left them as `px` values for now